e2e/scripts/post-snapshot.sql

```diff
@@ -0,0 +1,139 @@
+\set VERBOSITY verbose
```

Comment:
Can you add a comment to this file explaining what it does? I _assume_ that the intention is to try and insert a bunch of data, and see that it's possible to do so.

Reply:
yes i will, your assumption is correct

Reply:
done in [090040d](https://github.com/timescale/promscale_extension/pull/185/commits/090040d10bc921ac7dfde28f06e3f210ccfa0095)